src/CivicrmEntityListBuilder.php

```diff
@@ -23,9 +26,31 @@ public function buildHeader() {
   public function buildRow(EntityInterface $entity) {
     return [
       'id' => $entity->id(),
-      'label' => $entity->label(),
+      'label' => $entity->toLink(),
     ] + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getDefaultOperations(EntityInterface $entity) {
+    $operations = parent::getDefaultOperations($entity);
+
+    $operations['view'] = [
+      'title' => $this->t('View'),
+      'weight' => 50,
+      'url' => $this->ensureDestination($entity->toUrl()),
+    ];
+
+    return $operations;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function ensureDestination(Url $url) {
+    // @todo Disabled redirection to collection until paging fixed.
```

Comment:
I believe it was fixed and we can remove this.

Reply:
The destination is still not the most usable - I think there's more we could do here. But that doesn't mean we need a `@todo` :-)